connector/pom.xml

```diff
@@ -29,6 +35,11 @@
             <version>2.10.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.java-websocket</groupId>
+            <artifactId>Java-WebSocket</artifactId>
+            <version>1.5.1</version>
```

Comment:
Should probably specify compile scope as that's consistent with the rest of the code.